src/components/SearchBar.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./SearchBar.css";
+
+export default function SearchForm({
+  onHandleFormSubmit,
+  onHandleSearchInputChange
+}) {
+  return (
+    <form onSubmit={onHandleFormSubmit}>
+      <input
+        type='text'
+        name='city'
+        onChange={onHandleSearchInputChange}
+        placeholder='Search City...'
+        style={{ margin: "10px", padding: "5px" }}
+      />
+      <button>Search</button>
+    </form>
+  );
+}
+
+/*
+ <input type='submit' className='submit' />
```

Comment:
Are you not using this code anymore, remove it to keep your code clean.

Reply:
Sorry, I completely forgot :)